apps/twr_tag_tdma/pkg.yml

```diff
@@ -32,9 +32,11 @@ pkg.deps:
     - "@apache-mynewt-core/sys/shell"
     - "@apache-mynewt-core/sys/log/full"
     - "@apache-mynewt-core/sys/stats/full"
-#    - "@apache-mynewt-core/encoding/json"
+#   - "@apache-mynewt-core/encoding/json"
 #   - "@apache-mynewt-core/mgmt/imgmgr"
-    - "@apache-mynewt-core/mgmt/newtmgr"
+#   - "@apache-mynewt-core/mgmt/newtmgr"
```

Comment:
Don't leave the old newtmgr here as a comment, just remove it.

Reply:
Done